deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +204,62 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payloadEventsString: __ENV.DATA_BTM_SQS_PAYLOAD_EVENTS,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
 }
 
+const payloadEventsArray = JSON.parse(eventData.payloadEventsString)
+
+const payloadTimestampArray = eventData.payloadTimestamp.split(',')
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
```

Comment:
Could add these values to the test script itself instead of using an environment variable for this

Reply:
Current we want to test across a range of timescales for searching

Reply:
Will update the documentation on how to set the env var